nasa/harmony
# HARMONY-1562: Get Giovanni adapter working end to end

## Jira Issue ID
HARMONY-1562

## Description
Fixes the Giovanni adapter to work end to end.

## Local Test Steps
Pre-req: Make sure you have the giovanni-adapter included in your `LOCALLY_DEPLOYED_SERVICES`.

1. `npm run build-all-services`
2. `bin/restart-services`
3. http://localhost:3000/C1225808238-GES_DISC/ogc-api-coverages/1.0.0/collections/Grid%2FprecipitationCal/coverage/rangeset?format=text%2Fcsv&point=0.76,-3.8&subset=time(%222020-01-06T12%3A00%3A00Z%22%3A%222020-01-06T16%3A00%3A00Z%22)
4. Click on the job status URL
5. Verify the request is successful and includes a Giovanni link.
6. Click the Giovanni link and log in with EDL if not already logged in.
7. Verify you are redirected to a Giovanni output with precipitation data.

## PR Acceptance Checklist
